src/test/java/br/com/six2six/fixturefactory/transformer/DateTimeTransformerTest.java

```diff
@@ -0,0 +1,111 @@
+package br.com.six2six.fixturefactory.transformer;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.OffsetDateTime;
+import java.time.OffsetTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.Calendar;
+
+import javax.xml.datatype.DatatypeConfigurationException;
+
+import org.junit.Test;
+
+public class DateTimeTransformerTest {
+
+	@Test
+	public void transformCalendarToLocalDateTime() {
+		LocalDateTime date = LocalDateTime.now();
+		Calendar value = Calendar.getInstance();
+		value.setTimeInMillis(date.atZone(ZoneId.systemDefault()).toInstant().toEpochMilli());
+		
+		LocalDateTime transform = new DateTimeTransformer().transform(value, LocalDateTime.class);
+
+		assertNotNull("LocalDateTime should not be null", transform);
+		assertEquals("LocalDateTimes should be equal", date, transform);
+	}
+
+	@Test
+	public void transformCalendarToLocalDate() throws DatatypeConfigurationException {
+		LocalDate date = LocalDate.now();
+		Calendar value = Calendar.getInstance();
```

Comment:
I guess you forgot to change the `value` to represent the same date as `date`... otherwise you might get the different dates.

The same goes for the other methods down below.